tests/WE2E/test_configs/wflow_features/config.prdgen.yaml

```diff
@@ -0,0 +1,43 @@
+metadata:
+  description: |-
+    This test is to ensure that the product generation job
+    completes successfully on the RRFS_NA_3km grid using the RRFS_v1beta
+    physics suite with ICs and LBCs derived from the FV3GFS.
+    Note that this test also sets various resource parameters for several
+    of the rocoto tasks in order to more efficiently run the code on this
+    (very large) grid.
+user:
+  RUN_ENVIR: community
+workflow:
+  CCPP_PHYS_SUITE: FV3_RRFS_v1beta
+  PREDEF_GRID_NAME: RRFS_NA_3km
```

Comment:
That's an expensive test! Could we leverage any of the existing tests (if we already have a NA domain test) to also test this task? Could we alternatively test just the prodgen task with staged data?

Reply:
There is one WE2E test for the RRFS_NA_3km grid in the grids_extrn_mdls_suites_community directory.  It is called grid_RRFS_NA_3km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta, so it is very similar to the one I added here.  I'm willing to add the prdgen job to this pre-existing WE2E test if that makes sense and I can remove this one.  What do you think?

Reply:
I added the prdgen job to the pre-existing WE2E task for the RRFS_NA_3km grid.